lib/dialects/mssql/query.js

```diff
@@ -30,7 +31,11 @@ Query.prototype.run = function(sql, parameters) {
   var self = this;
   this.sql = sql;
 
-  this.sequelize.log('Executing (' + (this.connection.uuid || 'default') + '): ' + this.sql, this.options);
+  if (!(this.sequelize.options.benchmark || this.options.benchmark)) {
```

Comment:
Since the if is used twice maybe just normalize to a single variable.

`var benchmark = this.sequelize.options.benchmark || this.options.benchmark;` and use that.